### DISCUSSION TITLE
Dynamic background and info entities layout

### DISCUSSION THREAD

**Discussion by Jorre05** [marcokreeft87/room-card#101] [Q&A] [answered]
Hi,

I have two questions.
1) Is there a way to always show the info_entities below the tittle? Now, when the card is narrow, the info_entities values overlap with the title

2) Is there a wy to make the background image dynamic, based on the main entity state?

I've tried the code below, but that doesn't work:
<pre>
type: custom:room-card
title: Bureau
entity: light.licht_bureau_luchter
icon: mdi:sofa
show_icon: false
tap_action:
  action: navigate
  navigation_path: sub_bureau
styles:
  template:
    styles: >
      if (entity.state == 'off') return 'background-image: url(some image)'
      else return 'background-image: url(some other image)';
entities:
....
</pre>

Thanks

**Comment by marcokreeft87** [comment]
1. Not currently no, but a good idea!
2.  That should work, out of the top of my head, but you can check the entity html with developer tools to check what is set on the entity for css

**Comment by Jorre05** [comment]
I've tried several template/style code variations but none seem to work. There is no style code inserted in the resulting html.

**Comment by marcokreeft87** [comment]
Ah now that I have time to look at it on my laptop I see the problem. The main entity has no template support yet.
But thats a cool idea :) Im adding it to the todo list.

Its going to be a big update so it could take a while.

**Comment by marcokreeft87** [comment, marked as answer]
@Jorre05 check v1.07.00

**Comment by Jorre05** [comment]
Yes!, that works. Thank's a lot.

Now, another small thing...
On a mobile device, or any other device with a touch screen, the 'tap_action' fires when I start scrolling on an icon with an action defined. Is there any workaround for that?

**Reply by marcokreeft87** [reply]
Not really, you can try using hold action

**Reply by Jorre05** [reply]
That's what I did. Thanks again for the update.
